packages/graphql-playground-middleware-restify/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "graphql-playground-middleware-restify",
+  "version": "1.3.6",
+  "homepage":
+    "https://github.com/graphcool/graphql-playground/tree/master/packages/graphql-playground-middleware-restify",
+  "description":
+    "GraphQL IDE for better development workflows (GraphQL Subscriptions, interactive docs & collaboration).",
+  "contributors": [
+    "Tim Suchanek <tim@graph.cool>",
+    "Johannes Schickling <johannes@graph.cool>",
+    "Mohammad Rajabifard <mo.rajbi@gmail.com>"
+  ],
+  "repository": "http://github.com/graphcool/graphql-playground.git",
+  "license": "SEE LICENSE IN LICENSE",
+  "main": "dist/index.js",
+  "files": ["dist"],
+  "scripts": {
+    "build": "tsc",
+    "prepublishOnly": "npm run build"
+  },
+  "keywords": ["graphql", "graphiql", "playground", "graphcool"],
+  "peerDependencies": {
+    "restify": "^6.3.4"
+  },
+  "devDependencies": {
+    "@types/node": "^8.0.47",
+    "@types/restify": "^5.0.6",
+    "typescript": "^2.6.1"
+  },
+  "dependencies": {
+    "graphql-playground-html": "^1.3.6",
+    "graphql-playground-middleware": "^1.2.1-beta.6"
```

Comment:
please remove the `graphql-playground-middleware` dependency and update `graphql-playground-html` to the latest version